main.go

```diff
@@ -92,7 +92,8 @@ type prometheus struct {
 func NewPrometheus() *prometheus {
 	conf, err := config.LoadFromFile(*configFile)
 	if err != nil {
-		glog.Errorf("Couldn't load configuration (-config.file=%s): %v\n", *configFile, err)
+		glog.Errorf("Couldn't load configuration (-config.file=%s): %v", *configFile, err)
+		glog.Errorf("Note: The configuration format has changed with version 0.14, please check the documentation.")
```

Comment:
Once we have one :)

On Tue, May 5, 2015 at 4:34 PM Julius Volz <notifications@github.com> wrote:

> In main.go
> <https://github.com/prometheus/prometheus/pull/669#discussion_r29673640>:
>
> > @@ -92,7 +92,8 @@ type prometheus struct {
> >  func NewPrometheus() *prometheus {
> >  	conf, err := config.LoadFromFile(*configFile)
> >  	if err != nil {
> > -		glog.Errorf("Couldn't load configuration (-config.file=%s): %v\n", *configFile, err)
> > +		glog.Errorf("Couldn't load configuration (-config.file=%s): %v", *configFile, err)
> > +		glog.Errorf("Note: The configuration format has changed with version 0.14, please check the documentation.")
>
> This really needs a link to a migration doc :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/prometheus/pull/669/files#r29673640>.
>